pages.de/common/clang-format.md

```diff
@@ -0,0 +1,24 @@
+# clang-format
+
+> Programm zum Auto-Formatieren von C/C++/Java/JavaScript/Objective-C/Protobuf/C#-Code.
+> Weitere Informationen: <https://clang.llvm.org/docs/ClangFormat.html>.
+
+- Formatiere eine Datei und schreibe das Ergebnis nach stdout:
+
+`clang-format {{pfad/zu/quelldatei.cpp}}`
```

Comment:
I'm sorry, I don't think I understand. Do you want me to change the example filenames of the clang* commands across all translations or why should the PR be separate?

Reply:
I think what @pixelcmtd is trying to get at is that if you add `.cpp` on the end here, then it would be nice if you could contribute this back to the original english version of the page too :-)